shared/rakefiles/xk_util.rake

```diff
@@ -127,6 +127,58 @@ task :display_cluster_state => [:configure, :configure_secrets, :set_secrets] do
   end
 end
 
+task :display_universal_image_info => [:configure] do
+  sh "#{@exekube_cmd} sh -c ' \
+    UNIVERSAL_CI_URL=\"https://ci.gpii.net\";
+    UNIVERSAL_REPO=\"https://github.com/gpii/universal\";
+    RELEASE_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master-release\";
+    UPSTREAM_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master\";
+    LOOKUP_BUILDS=\"20\";
+
+    PREFERENCES_IMAGE_SHA=$(kubectl -n gpii get deployment preferences -o json 2> /dev/null | jq -r \".spec.template.spec.containers[0].image\" | grep -o \"sha256:.*\");
+    if [ \"$?\" != \"0\" ]; then
+      echo
```

Comment:
Since I'm looking at this code again, why do most lines end with `;` but some do not?

Reply:
`sh` collapses everything into a single string, `;` are required to separate commands that need it from each other. Alternative would be to escape newlines. I think this is another sign that bash code does not belong to rake and we should do something about it in future.

Reply:
If this is true, then does line 143 combine with line 144 and become `echo exit 1` (which will not exit)?

I guess if it worked in testing then the inconsistency isn't a big deal.